site/_includes/header.html

```diff
@@ -98,7 +98,7 @@
                       <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Documentation</a>
                       <ul class="dropdown-menu">
                         <li><a href="docs/home">Documentation Home</a></li>
-                        <li><a href="{{ site.projurl }}/docs/javadoc/index.html">Javadoc</a></li>
```

Comment:
I'm not really familiar with all of this so I'm willing to trust your judgement :-)

I think you understand the overall goal that I was hoping to achieve was to eliminate direct use of "Quarks" in general text (e.g., in md files) and "quarks" in URLs.  
But maybe it's not completely possible or practical or worth the effort as one can create a script that walks the files and changes "Quarks" and "quarks" to whatever the new values become.